<def f='boost-1.85.0/libs/contract/include/boost/contract/core/exception.hpp' l='630' ll='636' type='void boost::contract::postcondition_failure(boost::contract::from where)'/>
<doc f='boost-1.85.0/libs/contract/include/boost/contract/core/exception.hpp' l='614'>/**
Call failure handler for postconditions.

This is often called only internally by this library.

@b Throws:  This can throw in case programmers specify a failure handler that
            throws exceptions on contract assertion failures (not the default).

@param where    Operation that failed the contract assertion (e.g., this might
                be useful to program failure handler functors that never throw
                from destructors, not even when they are programmed by users to
                throw exceptions instead of terminating the program).

@see    @RefSect{advanced.throw_on_failures__and__noexcept__, Throw on Failure},
        @RefSect{tutorial.postconditions, Postconditions}
*/</doc>
<doc f='boost-1.85.0/libs/contract/include/boost/contract/core/exception.hpp' l='630'>/* can throw */</doc>
